hack/wait-for-namespace.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+##############################################################################
+# wait-for-namespace.sh
+#
+##############################################################################
+set -e
+
+if [ $# -eq 0 ]
+  then
+    >&2 echo "No arguments supplied"
+    exit 1
+fi
+
+# process command line args
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -n|--namespace)
+      shift;
+      NAMESPACES=( "$@" )
+      break;
+      ;;
+    -h|--help)
+      cat <<HELPMSG
+Valid command line arguments:
+  -n|--namespaces <name>: all of the namespaces we want to patch operator and wait for
+  -h|--help : this message
+HELPMSG
+      exit 1
+      ;;
+    *)
+      echo "Unknown argument [$key]. Aborting."
+      exit 1
+      ;;
+  esac
+done
+
+IS_MAISTRA=$([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true" || echo "false")
+
+if [ "${IS_MAISTRA}" == "false" ]; then
+  KIALI_CR_NAMESPACE_NAME="$(oc get kiali --all-namespaces -o custom-columns=NS:.metadata.namespace,N:.metadata.name --no-headers | sed 's/  */:/g')"
```

Comment:
You can ignore this but it's often easier to use jsonpath and/or jq to parse this stuff than it is to chop up the text e.g. `oc get kiali --all-namespaces -o jsonpath='{.items[*].metadata.namespace}'`